app/shared/types/settingsSchema.ts

```diff
@@ -133,7 +133,14 @@ export const settingsSchema = {
       type: 'object',
       properties: {
         _id: { type: 'string' },
-        semanticSearch: { type: 'boolean' },
+        tocGeneration: {
+          type: 'object',
+          required: ['active', 'url'],
```

Comment:
Is this right? You have a required property that is not defined in properties?

Reply:
this is not right, removed active from required 👍